modules/gapi/include/opencv2/gapi/media.hpp

```diff
@@ -30,6 +31,8 @@ class GAPI_EXPORTS MediaFrame {
     View access(Access) const;
     cv::GFrameDesc desc() const;
 
+    cv::util::any blobParams() const;
```

Comment:
What I'd add here is FIXME to design a better solution

Reply:
(we need to mark this portion of API unstable somehow)

Reply:
Added a `FIXME` along with some details on the behavior. Not sure about the unstable part though